ga4gh_tool_registry/validate.py

```diff
@@ -7,41 +7,20 @@
 
 from . import swg2salad
 import ruamel.yaml as yaml
+import warnings as warnings
 import requests
 from schema_salad.schema import load_schema, validate_doc
 from schema_salad import jsonld_context
 from schema_salad.main import printrdf
+from schema_salad.sourceline import cmap
 from schema_salad.ref_resolver import Loader
+from ruamel.yaml.comments import CommentedSeq, CommentedMap
 
 from cwltool.load_tool import validate_document
 
 from rdflib import Graph, URIRef, Literal, Namespace
 from rdflib.namespace import Namespace
 
-def dockstore_fixup(r):
-    if isinstance(r, list):
-        for d in r:
-            dockstore_fixup(d)
-    elif isinstance(r, dict):
-        if 'id' in r and r['id'] is None:
-            r['id'] = ''
-
-        if 'author' in r and r['author'] is None:
-            r['author'] = ''
-
-        if 'descriptor' in r and r['descriptor'] is None:
-            if 'dockerfile' in r:
-                r['descriptor'] = {"descriptor": "", "type": ""}
-            else:
-                r['descriptor'] = ''
-
-        if 'tooltype' in r:
-            r["toolclass"] = r['tooltype']
-            del r['tooltype']
-
-        for d in r.values():
-            dockstore_fixup(d)
-
```

Comment:
Not really important but PEP 8 expect 2 blank lines.  found 1

Reply:
I think we can probably fix this when we get CI running on this repo, thx!

Reply:
Actually, @garyluu  do you mind fixing this and adding a rudimentary travis.ci for running PEP? Thanks!